charts/policy-hub/README.md

```diff
@@ -6,12 +6,19 @@ For further information please refer to [Technical Documentation](../../docs/tec
 
 The referenced container images are for demonstration purposes only.
 
+## Prerequisites
+
+- [Kubernetes](https://kubernetes.io) 1.25.11+
+- [Helm](https://helm.sh) 3.9.3+
+- PV provisioner support in the underlying infrastructure
+
 ## Installation
 
 To install the chart with the release name `policy-hub`:
 
 ```shell
 $ helm repo add tractusx-dev https://eclipse-tractusx.github.io/charts/dev
+$ helm dependency update
```

Comment:
When installing a released chart from a chart repository (which is assumed here), it doesn't make sense to perform a `helm dependency update`, could you please remove that line?
`helm dependency update` needs to be executed if you still need to the download the dependencies of the Chart.yaml, which is necessary if you install the chart from locally checked out repository.

Reply:
A wasn't aware of that. Thanks for clarifying :) Fixed.